src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/Helper.java

```diff
@@ -235,6 +238,44 @@ public static void midRespondentECC(CaseData caseData, CaseData originalCaseData
         return errors;
     }
 
+    public static List<String> validateHearingDatesNotInFuture(CaseData caseData) {
```

Comment:
Why did you decide to add code concerned with case validation to a class called Helper?